.github/workflows/grafana-bench.yml

```
@@ -39,30 +41,21 @@ jobs:
         with:
           compose-file: './docker-compose.yml'
 
-      - name: Wait for Grafana to start
+      - name: Ensure Grafana is running
         run: |
           curl http://localhost:3000
```

```suggestion
      - name: Wait for Grafana to start
        run: |
          timeout 180 bash -c 'until curl -s -o /dev/null -w "%{http_code}" http://localhost:3000 | grep -q "200"; do sleep 1; done && echo "Grafana is up!"'
```

This would allow us to guard against Grafana not being fully ready to server requests before proceeding to the next step. Wdyt?

---

It's fine for now- honestly I think this is or should be built into bench. I need to review our docs and we can remove this step entirely